starters/gatsby-wordpress-starter/__tests__/snapshotTests/__snapshots__/pagination.test.jsx.snap

```diff
@@ -0,0 +1,1354 @@
+                      class="flex flex-col justify-center text-center no-underline
+          h-16 w-12 border-t-2 border-b-2 border-black bg-white hover:bg-blue-300 focus:bg-blue-200 focus:border-blue-300 border-blue-700 border-2
+          "
+                      href="/examples/pagination/1"
+                      state="[object Object]"
+                    >
+                      1
+                    </a>
+                  </li>
+                  <li>
+                    <a
+                      aria-current="false"
+                      aria-label="Go to Page 2"
+                      class="flex flex-col justify-center text-center no-underline
+          h-16 w-12 border-t-2 border-b-2 border-black bg-white hover:bg-blue-300 focus:bg-blue-200 focus:border-blue-300 false
+          "
+                      href="/examples/pagination/2"
+                      state="[object Object]"
+                    >
+                      2
+                    </a>
+                  </li>
+                  <li>
+                    <a
+                      aria-current="false"
+                      aria-label="Go to Page 3"
+                      class="flex flex-col justify-center text-center no-underline
+          h-16 w-12 border-t-2 border-b-2 border-black bg-white hover:bg-blue-300 focus:bg-blue-200 focus:border-blue-300 false
+          "
+                      href="/examples/pagination/3"
+                      state="[object Object]"
+                    >
+                      3
+                    </a>
+                  </li>
+                  <li>
+                    <a
+                      aria-current="false"
+                      aria-label="Go to Page 4"
+                      class="flex flex-col justify-center text-center no-underline
+          h-16 w-12 border-t-2 border-b-2 border-black bg-white hover:bg-blue-300 focus:bg-blue-200 focus:border-blue-300 false
+          "
+                      href="/examples/pagination/4"
+                      state="[object Object]"
+                    >
+                      4
+                    </a>
+                  </li>
+                  <li>
+                    <a
+                      aria-current="false"
+                      aria-label="Go to Page 5"
+                      class="flex flex-col justify-center text-center no-underline
+          h-16 w-12 border-t-2 border-b-2 border-black bg-white hover:bg-blue-300 focus:bg-blue-200 focus:border-blue-300 false
+          "
+                      href="/examples/pagination/5"
+                      state="[object Object]"
+                    >
+                      5
+                    </a>
+                  </li>
+                  <li>
+                    <button
+                      aria-label="Expand Hidden Buttons"
+                      class="hidden md:block h-16 w-12 border-2 border-black bg-slate-200 hover:bg-blue-300 focus:bg-blue-200 focus:border-blue-300\\"
+				}"
```

Comment:
Curious about this `\\` and `}"`. Also noticing the state for each link is rendering as `"[object Object]"`

Reply:
Ah it seems the `\\` and `}"` slipped in from a styling mistake currently in the `nextjs-kit` paginator. Will update in both places.